youtube_dl/extractor/acast.py

```diff
@@ -93,7 +96,7 @@ class ACastChannelIE(ACastBaseIE):
     _VALID_URL = r'''(?x)
                     https?://
                         (?:
-                            (?:www\.)?acast\.com/|
+                            (?:www|shows\.)?acast\.com/|
```

Comment:
You need to keep the `.` match for both domain prefixes, like l.51
```suggestion
                            (?:(?:www|shows)\.)?acast\.com/|
```